pkg/networkservice/kernelutils/common.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 SUSE LLC.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
A lot of shameless copy/paste in this file, and a lot of unused code here too.

Reply:
Don’t forget to carry forward the original copyright in addition to the SUSE copyright for copied code